hw11.php

```diff
@@ -0,0 +1,242 @@
+<?php
+
+
+
+// task 1
+//$fruits = ["Apple", "Banana", "Orange"];
+//echo count($fruits);
+
+
+
+// task 2
+//$arr = [
+//    'name' => 'Nikita',
+//    'surname' => 'Gonchar',
+//    'age' => 20
+//];
+//echo $arr['name'];
+
+
+
+// task 3
+//$message = 'Hello';
+//echo $message[0], $message[1], $message[4];
+
+
+
+// task 4
+//$arr = [1, 2, 3, 4, 5];
+//$x = arr(1, 2, 3, 4, 5);
```

Comment:
эм, это даже не работает
```suggestion
//$x = array(1, 2, 3, 4, 5);
```